app/src/main/java/com/nutomic/syncthingandroid/views/FoldersAdapter.java

```diff
@@ -132,16 +136,16 @@ private static String getLocalizedState(Context c, String state, int percentage)
     }
 
     /**
-     * Requests updated model info from the api for all visible items.
+     * Requests updated folder status from the api for all visible items.
      */
-    public void updateModel(RestApi api) {
+    public void updateFolderStatus(RestApi api) {
         for (int i = 0; i < getCount(); i++) {
-            api.getModel(getItem(i).id, this::onReceiveModel);
+            api.getFolderStatus(getItem(i).id, this::onReceiveFolderStatus);
         }
     }
 
-    private void onReceiveModel(String folderId, Model model) {
-        mModels.put(folderId, model);
+    private void onReceiveFolderStatus(String folderId, FolderStatus FolderStatusInfo) {
```

Comment:
Copy pasta variable naming

Reply:
Indeed, sorry. I'll update that. Thanks for your patience.